src/css/_TextSelection.scss

```diff
@@ -0,0 +1,41 @@
+.textSelectionSVG {
+    width: 100%;
+    position: absolute;
+    height: 100%;
+    top: 0;
+    left: 0;
+    pointer-events: none;
+
+    .BRwordElement {
+        pointer-events: all;
+    }
```

Comment:
```suggestion

    // Make it so right-clicking on "blank" part of svg sends events to the image (for saving)
    pointer-events: none;
    .BRwordElement { pointer-events: all; }
```

Reply:
Bug: clicking/dragging on a word, but mouse-uping over a blank area no longer works. We need the mouseup listener to be attached do `$(document)` maybe?